web/cobrands/fixmystreet/staff.js

```diff
@@ -340,5 +340,29 @@ $.extend(fixmystreet.set_up, {
             $input.val($this.val());
         }
     });
+  },
+
+  update_problem_fields: function(role, body, args) {
```

Comment:
This should not be in `fixmystreet.set_up` as you do not want it to run at startup, which all functions in `set_up` are. That's why I suggested its own hooks object.